apps/govquests-api/rails_app/db/seeds.rb

```diff
@@ -105,6 +119,18 @@ def create_action(action_data)
     quest_type: "Governance",
     audience: "Delegates",
     rewards: [{type: "Points", amount: 100}],
+    actions: [action_ids[4]]
+  },
+  {
+    display_data: {
+      title: "Top 100 Delegates",
+      intro: "Have you reached the Top 100 Delegates in Season 6? Let’s check your ranking and, if you’re among the top, celebrate your achievement with a reward!",
+      image_url: "https://example.com/advanced-governance.jpg",
+      requirements: 'You need to be a delegate to do this quest! If you’re not one, start with Become Delegate Quest.'
+    },
+    quest_type: "Governance",
+    audience: "Delegates",
+    rewards: [{type: "Points", amount: 100}],
```

Comment:
👍 I refactored the seed to be a bit more readable and already merged that code. lmk if you need any help refactoring yours to that format

Reply:
@ribeirojose Nice refactor on seed, I had no problem to refactor to that format!